lib/libevent/libevent.nit

```diff
@@ -458,6 +477,38 @@ extern class ConnectionListener `{ struct evconnlistener * `}
 	end
 end
 
+# A listener acting on an Unix Socket, spawns `Connection` on new connections
+extern class UnixConnectionListener `{ struct evconnlistener * `}
+
+	super ConnectionListener
+
+	private new bind_to(base: NativeEventBase, address: CString, file: CString, factory: ConnectionFactory)
+	import ConnectionFactory.accept_unix_connection, error_callback `{
+		struct sockaddr_un sun;
+		struct evconnlistener *listener;
+
+		printf("Creating socket on %s\n",file );
```

Comment:
I'm not a big fan of printing stuff in a library, if this is a debug print, it should be removed in my opinion

Reply:
You're completely right ! I had so much debugging stuff in my local version I forgot to remove this one ^^